index.html

```diff
@@ -19,7 +19,11 @@ <h1>Welcome to the student showcase!</h1>
     </div>
     <div class="content">
         <div>ONE</div>
-        <div>TWO</div>
+    <div img src="media/me.jpg" alt="Selfie">Allan Caprola
```

Comment:
`<div>` and `<img>` are separate elements, and should be treated as a parent `div` that wraps the rest of your content.